src/scripts/package.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+set -e
+set -o nounset
+set -o errexit
+set -o xtrace
+
+cd ${PROJECT_DIR} || exit
+zip -r ${PACKAGE_PATH} . -x "node_modules/*" "lib/script/*" ".git/*" ".circleci/*"
```

Comment:
existence check is done for `PROJECT_DIR` and then `PACKAGE_PATH` is used, intended?

Reply:
sry, didn't look correctly